sass/pages/_news.scss

```diff
@@ -23,6 +23,25 @@
         border-radius: $border-radius;
         max-width: 100%;
     }
+
+    table {
+        border-spacing: 0;
+        border: 2px solid $subtitle-color;
```

Comment:
Using `$subtitle-color` is a bit strange here, but there isn't an existing color variable that would work